src/routes/Router.tsx

```diff
@@ -1,19 +1,19 @@
-import {HashRouter, Route, Redirect, Switch} from "react-router-dom";
+import {Route, Redirect, Switch, BrowserRouter} from "react-router-dom";
 import {FunctionComponent} from "react";
 import LoginBoard from "routes/LoginBoard/LoginBoard";
 import NewBoard from "routes/NewBoard/NewBoard";
 import BoardGuard from "routes/Board/BoardGuard";
 import PrivateRoute from "routes/PrivateRoute";
 
 const Router = () => (
-  <HashRouter>
+  <BrowserRouter>
```

Comment:
BrowserRouter will lead to different paths that have to be configured in our deployment